{"uuid": "95712363-b681-4583-90f5-bd2e4ce8bdfd", "children": ["f9ba1be2-d1b1-47f5-88d9-3a6beb3703a1"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "AttributeError: 'function' object has no attribute 'get'\n", "trace": "  File \"/Users/chethangopal/Desktop/Learning/.venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/Users/chethangopal/Desktop/Learning/.venv/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/Users/chethangopal/Desktop/Learning/.venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/chethangopal/Desktop/Learning/.venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1122, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/chethangopal/Desktop/Learning/.venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 877, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/Users/chethangopal/Desktop/Learning/tests/conftest.py\", line 29, in driver\n    driver.get(base_url)\n    ^^^^^^^^^^\n"}, "start": 1716712741368, "stop": 1716712741368}], "start": 1716712741368, "stop": 1716712741370}